src/index.js

```diff
@@ -39,8 +41,7 @@ exports.unmarshalPublicKey = (buf) => {
 exports.marshalPublicKey = (key, type) => {
   type = (type || 'rsa').toLowerCase()
 
-  // for now only rsa is supported
-  if (type !== 'rsa') {
+  if (type !== 'rsa' && type !== 'ed25519') {
```

Comment:
Support is not great yet for `Array.prototype.includes` I think, but feel free to add `lodash.includes`which we use in other places in our modules.